src/modules/github/components/pull-list-by-repo/pull-list-by-repo.tsx

```diff
@@ -28,38 +32,30 @@ export const PullListByRepo: FC<PullListByRepoProps> = ({
   return (
     <div>
       <div className="flex">
-        <input
-          type="checkbox"
+        <Checkbox
           checked={isRepoChecked}
-          onChange={() => handleRepoChange(repo)}
+          onCheckedChange={() => handleRepoChange(repo)}
           disabled={!hasChild}
         />
-        <span className="ml-2">{repo}</span>
-        {hasChild && (
-          <svg
-            xmlns="http://www.w3.org/2000/svg"
-            width="10"
-            height="10"
-            viewBox="0 0 100 100"
-            className={'ml-2' + (isRepoOpen ? ' rotate-180' : ' rotate-90')}
-            onClick={handleRepoClick}
-          >
-            <polygon points="50,0 100,100 0,100" fill="black" />
-          </svg>
-        )}
+        <Label className="ml-2">{repo}</Label>
+        {hasChild &&
+          (isRepoOpen ? (
+            <ChevronDownIcon className={'ml-2'} onClick={handleRepoClick} />
+          ) : (
+            <ChevronRightIcon className={'ml-2'} onClick={handleRepoClick} />
```

Comment:
there is no reason to add a curly bracket here. you can just use like `className="ml-2"`